### DISCUSSION TITLE
Want a add a cursor for snippets like adding $ symbols for multiple cursors

### DISCUSSION THREAD

**Discussion by MdNihal05** [cpeditor/cpeditor#1203] [Q&A]
How can I add multiple cursors so that I can reach that position on tabs 
In snippets
Note : I code in c++

**Comment by ouuan** [comment]
There are no these features now. There is a request for auto-complete at #156.

**Reply by MdNihal05** [reply]
Updated:
Can you add this feature I mean can you work on this feature
